src/common/Defs.h

```diff
@@ -31,7 +31,11 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef COMMON_DEFS_H_
 #define COMMON_DEFS_H_
 
-#include "DaemonBuildInfo.h"
+#if defined(BUILD_VM)
+#include "DaemonBuildInfo/Game.h"
+#else
+#include "DaemonBuildInfo/Engine.h"
+#endif
```

Comment:
Do these include the same values? Should we keep the header the same and only swap out the c++ files with the variables?

Reply:
They may not set the same variables, that's why it broke the build. For example the game cmake doesn't set the linux engine OpenGL option so when building a game it was rewriting the files without such variable.